README.md

```diff
@@ -190,3 +190,36 @@ There are three optional parameters to ```forgae contracts```:
     It defaults to the following url -  http://localhost:3001/; 
 - --update - update the contracts aepp with the latest version;
 - --ignoreOpenInBrowser - ignoring opening of the browser;
+
+## deployed contract's instance has additional functionality
```

Comment:
I think that this is a bit confusing. Here you can specify what is this additional functionality. This section can be named something like this "Access deployed smart contract functions" for example. YOu can explain that they are now added to the deployed instance.